src/gwt/src/org/rstudio/core/client/widget/DataTableColumnWidget.java

```diff
@@ -0,0 +1,92 @@
+package org.rstudio.core.client.widget;
+
+import com.google.gwt.user.client.ui.TextBox;
+import com.google.gwt.event.dom.client.*;
```

Comment:
This was IntelliJ auto changing it to `*` because I incidentally did include everything inside the package. If I were not using everything in the package it would automatically change it to the correct includes. It's probably a setting I can change to prefer being verbose versus using `*` when it's not overly inclusive for line reduction. I'm fine with this either way.